airbyte-integrations/connector-templates/source-singer/setup.py.hbs

```diff
@@ -38,7 +38,7 @@ setup(
         # Dependencies required by the main package but not tests should go in main. Deps required by
         # integration tests but not the main package go in tests. Deps required by both should go in
         # install_requires.
-        "main": ["base-singer", "base-python"],
+        "main": ["{{ tap_name }}", "base-singer", "base-python"],
```

Comment:
can we get rid of the `main` package and put all this in `install_requires`?

Reply:
also, we should get the tap version from them so it doesn't change from underneath us

Reply:
to be clear my goal is not to make that part easier for a user. Just to be able to inject an existing tap to ensure our implementation continues to work